Test `test_storage_alias_replicated/test.py::test_alias_with_replicated` is flaky

https://play.clickhouse.com/play?user=play&run=1#V0lUSAogICAgOTAgQVMgaW50ZXJ2YWxfZGF5cwpTRUxFQ1QKICAgIHRvU3RhcnRPZkRheShjaGVja19zdGFydF90aW1lKSBBUyBkYXksCiAgICBjb3VudCgpIEFTIGZhaWx1cmVzLAogICAgZ3JvdXBVbmlxQXJyYXkocHVsbF9yZXF1ZXN0X251bWJlcikgQVMgcHJzLAogICAgYW55KHJlcG9ydF91cmwpIEFTIHJlcG9ydF91cmwKRlJPTSBjaGVja3MKV0hFUkUgKG5vdygpIC0gdG9JbnRlcnZhbERheShpbnRlcnZhbF9kYXlzKSkgPD0gY2hlY2tfc3RhcnRfdGltZQogICAgQU5EIHRlc3RfbmFtZSA9ICd0ZXN0X3N0b3JhZ2VfYWxpYXNfcmVwbGljYXRlZC90ZXN0LnB5Ojp0ZXN0X2FsaWFzX3dpdGhfcmVwbGljYXRlZCcKICAgIC0tIEFORCBjaGVja19uYW1lID0gJ0ludGVncmF0aW9uIHRlc3RzIChhbWRfYmluYXJ5LCA1LzUpJwogICAgQU5EIHRlc3Rfc3RhdHVzIElOICgnRkFJTCcsICdFUlJPUicpCiAgICBBTkQgKChoZWFkX3JlZiA9ICdtYXN0ZXInIEFORCBwdWxsX3JlcXVlc3RfbnVtYmVyID0gMCkgT1IgcHVsbF9yZXF1ZXN0X251bWJlciAhPSAwKQpHUk9VUCBCWSBkYXkKT1JERVIgQlkgZGF5IERFU0MK

Examples:
```
File: test_storage_alias_replicated/test.py:40 - in test_alias_with_replicated
    assert node2.query("SELECT * FROM test_rmt.rmt_table") == "1\tone\n"
E   AssertionError: assert '' == '1\tone\n'
```

```
File: test_storage_alias_replicated/test.py:52 - in test_alias_with_replicated
    assert "2\ttwo" in node1.query("SELECT id, value FROM test_rmt.rmt_table WHERE id = 2")
E   AssertionError: assert '2\ttwo' in ''
E    +  where '' = query('SELECT id, value FROM test_rmt.rmt_table WHERE id = 2')
E    +    where query = <helpers.cluster.ClickHouseInstance object at 0x7fc9104947f0>.query
```

@nauu @azat please take a look

--- a/tests/integration/test_storage_alias_replicated/test.py
+++ b/tests/integration/test_storage_alias_replicated/test.py
@@ -37,18 +37,19 @@ def test_alias_with_replicated(started_cluster):
 
     node1.query("INSERT INTO test_rmt.alias_rmt VALUES (1, 'one')")
 
+    node2.query("SYSTEM SYNC REPLICA test_rmt.rmt_table")
     assert node2.query("SELECT * FROM test_rmt.rmt_table") == "1\tone\n"
 
     node1.query("ALTER TABLE test_rmt.alias_rmt ADD COLUMN time DateTime DEFAULT now()")
 
-    node2.query("SYSTEM SYNC REPLICA test_rmt.rmt_table")
-
+    node2.query("SYSTEM SYNC DATABASE REPLICA test_rmt")
     assert "time" in node1.query("DESCRIBE test_rmt.alias_rmt")
     assert "time" in node2.query("DESCRIBE test_rmt.rmt_table")
     assert "time" in node2.query("DESCRIBE test_rmt.alias_rmt")
 
     node2.query("INSERT INTO test_rmt.alias_rmt VALUES (2, 'two', now())")
 
+    node1.query("SYSTEM SYNC REPLICA test_rmt.rmt_table")
     assert "2\ttwo" in node1.query("SELECT id, value FROM test_rmt.rmt_table WHERE id = 2")
     assert "2\ttwo" in node2.query("SELECT id, value FROM test_rmt.alias_rmt WHERE id = 2")
 

